mw-webapp/src/component/header/Header.tsx

```diff
@@ -39,12 +66,28 @@ export const Header = () => {
           onClick={user ? logOut : logIn}
           value={user ? BUTTON_LOG_OUT_VALUE : BUTTON_LOG_IN_VALUE}
         />
-        {/* //TODO: we need delete this link when after sidebar with all links to al pages will be added */}
-        <Link
-          path={pages.way.path("uuid")}
-          value={LINK_TEXT}
+        <Sidebar
+          trigger={
+            <Button
+              value={"Navigation"}
+              onClick={() => {}}
+            />
+          }
+          content={
+            <div className={styles.navSidebarContent}>
+              {navigationLinks.map((item) => (
+                item && (
+                  <Link
+                    key={item.value}
+                    path={item.path}
+                    value={item.value}
+                  />
+                )
+              ))}
+            </div>
```

Comment:
move function to top of file (before return). you can call it renderNavifationLinks

Reply:
done